setup.py

```diff
@@ -50,6 +51,8 @@
             'black<=19.10b0',
             'flake8<=3.9',
             'isort~=5.0',
+            'docutils~=0.16',
```

Comment:
`docutils` is a dependency of `readme-render`.Maybe we can skip adding that if it does not requires to be kept locked at a specific version.